### PR TITLE
Page Parameters in Studio

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

File: content/en/docs/studio/page-editor/page-editor-widgets/page-editor-data-view-list-view.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The data source determines which object will be shown in the data view. For gene
...
+| Context              | A data source determining that when the page is opened, it expects the selected object to be passed to it. Data views with this data source get the object from the context. You select an entity for this data source and data from this entity will be shown in the data view. Entities available on the page are defined by [page parameters](/studio/page-parameter/).  <br />For more technical information on the context source, see [Context Source](/refguide/context-source/) in the *Studio Pro Guide*. |
 | Microflow            | A data source that runs a selected microflow and displays a return value. For more technical information, see [Microflow Source](/refguide/microflow-source/) in the *Studio Pro Guide*. |
 | List widget          | A data source that allows a data view to display detailed information on an object in the list widget (list view) on the same page. For more technical information, see [Listen To Widget Source](/refguide/listen-to-grid-source/) in the *Studio Pro Guide*. |
```
Comment (arjan-h) [inline content/en/docs/studio/page-editor/page-editor-widgets/page-editor-data-view-list-view.md:44]: Note: the XPath option of List View (Par 3.1) should be removed. 
XPath and Data source are the same in Studio.

File: content/en/docs/studio/page-editor/page-editor-widgets/page-editor-data-view-list-view.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ The data source determines which object will be shown in the data view. For gene
...
-| Context              | A data source determining that when the page is opened, it expects the selected object to be passed to it. Data views with this data source get the object from the context. You select an entity for this data source and data from this entity will be shown in the data view. <br />If your data view is placed inside another data container, for example, inside a list view, you can also select an entity that has associations with the entity of this list view. <br />For more technical information on the context source, see [Context Source](/refguide/context-source/) in the *Studio Pro Guide*. |
+| Context              | A data source determining that when the page is opened, it expects the selected object to be passed to it. Data views with this data source get the object from the context. You select an entity for this data source and data from this entity will be shown in the data view. Entities available on the page are defined by [page parameters](/studio/page-parameter/).  <br />For more technical information on the context source, see [Context Source](/refguide/context-source/) in the *Studio Pro Guide*. |
```
Comment (arjan-h) [inline content/en/docs/studio/page-editor/page-editor-widgets/page-editor-data-view-list-view.md]: The new text is not entirely correct. The text you removed is still valid, but now you have Page parameters next to it

Comment (arjan-h) [inline content/en/docs/studio/page-editor/page-editor-widgets/page-editor-data-view-list-view.md]: I think Entities is not correct nor consistent in this context. Shouldnt we use 'Objects' instead

File: content/en/docs/studio/page-editor/page-parameter/_index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+title: "Page Parameter"
+url: /studio/page-parameter/
+description: "Describes page parameters in Mendix Studio."
+weight: 10
+tags: ["studio", "parameter", "page parameter", "page"]
+---
+
+## 1 Introduction 
+
+Page parameters provide context to data views and allow you using multiple objects that do not have an association. You can pass multiple objects when opening a page same way microflows do. For example, you can add multiple top-level data views, each connected to a different parameter and displaying data from their entities. Additionally, you can directly select associations of a page parameter.
```
Comment (arjan-h) [inline content/en/docs/studio/page-editor/page-parameter/_index.md]: I think a screenshot of the Parameter tray with,say, 2 parameters would help paint the picture

Comment (arjan-h) [inline content/en/docs/studio/page-editor/page-parameter/_index.md]: I would like to start with explaining params from the perspective of the page: input that needs to be passed from the calling page,mf,.... to the page. Information that can be used on the page again.

E.g. if you'd like to build an EmployeeDetailEdit page, then the page will have a parameter Employee. When calling the page a concrete Employee object needs to be passed in order to be able to show the right Employee content.

And then afterwards we could tell about data views and the fact they can use page parameters as their context

File: content/en/docs/studio/page-editor/page-parameter/_index.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+## 2 Properties
+
+Parameter consists of the following general properties:
+
+* [Entity](#entity)
+* [Name](#name)
+
+### 2.1 Entity {#entity}
+
+The entity of the parameter. When the page is opened, the selected object is passed to it. This object can be used by data views, i.e. data from this entity will be shown in a data view.
```
Comment (arjan-h) [inline content/en/docs/studio/page-editor/page-parameter/_index.md]: When the page is opened, the selected object is passed to it. ->
"When the page is opened, an object of this entity type should be passed to it."

File: content/en/docs/studio/page-editor/page-editor-widgets/page-editor-data-view-list-view.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ The data source determines which object will be shown in the data view. For gene
...
+| Context              | A data source determining that when the page is opened, it expects the selected object to be passed to it. Data views with this data source get the object from the context that is defined by [page parameters](/studio/page-parameter/). You select an entity for this data source and objects of this entity will be shown in the data view.  <br /><br />If your data view is placed inside another data container, for example, inside a list view, you can also select an entity that has associations with the entity of this list view. <br />For more technical information on the context source, see [Context Source](/refguide/context-source/) in the *Studio Pro Guide*. |
```
Comment (mlaponder) [inline content/en/docs/studio/page-editor/page-editor-widgets/page-editor-data-view-list-view.md]: As a data view will only show a single object should "objects of this entity" not be "object of this entity"?

File: content/en/docs/studio/page-editor/page-parameter/_index.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+## 1 Introduction 
+
+A page parameter is an input that needs to be passed from the calling page or a microflow to the current page. Page parameters determine information that can be re-used on the page. For example, if you would like to build an **Employee_Details_Edit** page, the page will have a parameter **Employee**. This means that when this page is called a corresponding Employee object needs to be passed to it to show the corresponding Employee data.
+
+Page parameters provide context to data views and allow you using multiple objects that do not have an association. 
+
+{{< figure src="/attachments/studio/page-editor/page-parameter/page-parameters.png" >}}
+
+You can pass multiple objects when opening a page same way microflows do. For example, you can add multiple top-level data views, each connected to a different parameter and displaying data from their entities: you can show Employee’s data in one data view and Order’s data in another data view adding two page parameters to the page.
```
Comment (mlaponder) [inline content/en/docs/studio/page-editor/page-parameter/_index.md]: Shouldn't it be "the same way microflows do" 🤔 ?

File: content/en/docs/studio/page-editor/page-editor-widgets/page-editor-data-view-list-view.md (hunk 5 above)
Comment (MariaShaposhnikova) [reply content/en/docs/studio/page-editor/page-editor-widgets/page-editor-data-view-list-view.md]: Good point! I changed it to singular.

**Pull request merged**